packages/api-page-builder-so-ddb-es/src/definitions/blockEntity.ts

```diff
@@ -0,0 +1,58 @@
+import { Entity, Table } from "dynamodb-toolbox";
+import { Attributes } from "~/types";
+
+interface Params {
+    table: Table;
+    entityName: string;
+    attributes: Attributes;
+}
+
+export const createBlockEntity = (params: Params): Entity<any> => {
+    const { entityName, attributes, table } = params;
+    return new Entity({
+        name: entityName,
+        table,
+        attributes: {
+            PK: {
+                partitionKey: true
+            },
+            SK: {
+                sortKey: true
+            },
+            TYPE: {
+                type: "string"
+            },
+            id: {
+                type: "string"
+            },
+            name: {
+                type: "string"
+            },
+            type: {
+                type: "string"
+            },
+            blockCategory: {
```

Comment:
yes, but I suggest to keep it as is.
We have a "Category" entity in Page Builder app (it describes a "page category"), so it may be confusing to use "category" here (As it is a string value of slug, so nothing is pointing that it is a "blockCategory", not a "category" of the pages)

Reply:
Alright, let's keep it as is. 

Thanks for clarification!